src/scenes/Activities/scenes/EventHistory.js

```diff
@@ -128,14 +131,13 @@ export default class EventHistory extends Component {
   }
 
   render() {
-    const { classes } = this.props;
     const { transactions } = this.state;
 
     return (
       <Grid container spacing={0}>
         {
           transactions.length ?
-            (<Table className={classes.historyTable}>
```

Comment:
don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

Reply:
uh, I remove because I didn't find it tbh.

Reply:
even back to https://github.com/bodhiproject/bodhi-ui/commit/bc501aefc1b25c22b5ad66a0f38fe8e2049763c3 I still didn't see the historyTable anywhere. 

if you use the page, nothing wrong to be seen now.

Reply:
even back to https://github.com/bodhiproject/bodhi-ui/commit/bc501aefc1b25c22b5ad66a0f38fe8e2049763c3 I still didn't see the `historyTable`, `clickToExpandRow`, `detailRow`, `viewEventLink ` anywhere. 

if you use the page, nothing wrong to be seen now.

Reply:
or can you point me to the original table dropdown design? I can check if those missing classess are necessary.